src/client/Microsoft.Identity.Client/Platforms/Features/DesktopOS/Kerberos/TicketCacheReader.cs

```diff
@@ -5,6 +5,7 @@
 using System.ComponentModel;
 using System.Runtime.InteropServices;
 using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+using static System.Net.WebRequestMethods;
```

Comment:
Please undo this.

Reply:
Yeah I am actually not sure why this was added. Will remove. Assuming over-eager IntelliSense/IntelliCode.